README.md

```diff
@@ -1,28 +1,28 @@
-## Heroku Stack Images
+## Heroku Base Images
 
-[![CI](https://github.com/heroku/stack-images/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/stack-images/actions/workflows/ci.yml)
+[![CI](https://github.com/heroku/base-images/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/base-images/actions/workflows/ci.yml)
 
-This repository holds recipes for building [Heroku stack images](https://devcenter.heroku.com/articles/stack).
+This repository holds recipes for building the base images for [Heroku stacks](https://devcenter.heroku.com/articles/stack).
 The recipes are also rendered into Docker images that are available on Docker Hub:
 
-| Image                                     | Base                                  | Type                       | Status      |
-|-------------------------------------------|---------------------------------------|----------------------------|-------------|
-| [heroku/heroku:18][heroku-tags]           | [ubuntu:18.04][ubuntu-tags]           | Heroku Runtime Stack Image | End-of-life |
-| [heroku/heroku:18-build][heroku-tags]     | [heroku/heroku:18][heroku-tags]       | Heroku Build Stack Image   | End-of-life |
-| [heroku/heroku:18-cnb][heroku-tags]       | [heroku/heroku:18][heroku-tags]       | CNB Runtime Stack Image    | End-of-life |
-| [heroku/heroku:18-cnb-build][heroku-tags] | [heroku/heroku:18-build][heroku-tags] | CNB Build Stack Image      | End-of-life |
-| [heroku/heroku:20][heroku-tags]           | [ubuntu:20.04][ubuntu-tags]           | Heroku Runtime Stack Image | Available   |
-| [heroku/heroku:20-build][heroku-tags]     | [heroku/heroku:20][heroku-tags]       | Heroku Build Stack Image   | Available   |
-| [heroku/heroku:20-cnb][heroku-tags]       | [heroku/heroku:20][heroku-tags]       | CNB Runtime Stack Image    | Available   |
-| [heroku/heroku:20-cnb-build][heroku-tags] | [heroku/heroku:20-build][heroku-tags] | CNB Build Stack Image      | Available   |
-| [heroku/heroku:22][heroku-tags]           | [ubuntu:22.04][ubuntu-tags]           | Heroku Runtime Stack Image | Recommended |
-| [heroku/heroku:22-build][heroku-tags]     | [heroku/heroku:22][heroku-tags]       | Heroku Build Stack Image   | Recommended |
-| [heroku/heroku:22-cnb][heroku-tags]       | [heroku/heroku:22][heroku-tags]       | CNB Runtime Stack Image    | Recommended |
-| [heroku/heroku:22-cnb-build][heroku-tags] | [heroku/heroku:22-build][heroku-tags] | CNB Build Stack Image      | Recommended |
+| Image                                     | Base                                  | Type                      | Status      |
+|-------------------------------------------|---------------------------------------|---------------------------|-------------|
+| [heroku/heroku:18][heroku-tags]           | [ubuntu:18.04][ubuntu-tags]           | Heroku Runtime Base Image | End-of-life |
```

Comment:
Here and in `BUILD.md` we use a few different variations of the build/run terms, some of which are inconsistent - for example some have hyphenation, some don't, and some have the "time" suffix, some don't.

i.e: It seems like we should use one of these three (and not mix and match):
1. "Build" vs "Run"
2. "Build-time" vs "Run-time"
3. "Buildtime" vs "Runtime"

(whereas at the moment in README.md we use "Runtime" vs "Build", and in Build.md we use "runtime" vs "build-time")

I think I'm leaning towards (1), given that:
- "Runtime" sounds like the "runtime" in "runtime team"
- The upstream CNB project calls them "Build" and "Run", per https://github.com/buildpacks/spec/blob/main/platform.md#terminology

Reply:
@edmorley made changes to `README.md` and `BUILD.md` to reflect (1). Expect those usages you noted to consistently use "Base Run Image" or "Base Build Image".